tailcfg/tka.go

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package tailcfg
+
+import (
+	"tailscale.com/types/key"
+	"tailscale.com/types/tkatype"
+)
+
+// TKAInitBeginRequest submits a genesis AUM to seed the creation of the
+// tailnet's key authority.
+type TKAInitBeginRequest struct {
+	// NodeID of the initiating client.
```

Comment:
complete sentences:

https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

This sentence lacks a verb. Maybe:

```
// NodeID is the node of the initiating client.
// It must match the machine key being used to communicate over noise.
```